scripts/train.py

```diff
@@ -57,6 +57,10 @@
                      'The number of cpus for each worker.')
 flags.DEFINE_bool('use_gpu', True, 'Whether to use gpu or not.')
 
+flags.DEFINE_integer(
```

Comment:
are the examples shuffled? to avoid that by some reason the first 1000 are all very different from the rest (e.g. they all have the same low wind speed)?

Reply:
The dataset is randomly generated and randomly stored. There is no reason for them not to be shuffled. BUt we can always just use `dataset.shuffle`  provided by the hugging face API